README.md

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
yeah, but my editor still hates on lines with trailing spaces, and i don't think I should stop it :)